Updates for workshop

markstur opened the pull request:
* Some config handling cleanup
* Split instructions to better cover DB cloud vs AWS and then App locally vs AWS
* Add initial raw API and page to access clickstream history so we can demonstrate accessing the clickstream
* The history part will be built upon for the workshop (or left todo for talking points)

markstur committed:
Improve env and connection handling and logging

* Better logging for config and connection
* Re-order clicks.service.js for demo (readability)
* Always try to create the table if it does not exist

markstur committed:
Organize/split the readme for DBaaS vs OpenShift

Reorganized so there is DB on OpenShift or IBM Cloud and then
start app on OpenShift or locally. Filled out more of the setup.

markstur committed:
History API and page to show users clickstream history

Adding history API/page to show how data can be used.
Raw for now.

markstur committed:
Log time spent on page (category)

Not in DB (yet) just logging, but added the ability
to track time spent on page. Just the category pages
for first test.

markstur committed:
Cleanup unused/commented-out lines in history page

rhagarty commented on doc/source/aws_app.md:
```diff
@@ -0,0 +1,87 @@
+* Hit the `Save` button
+* The application will restart with an environment that includes the environment variables that you set using a key/value secret.
+
+![create_secret](images/create_secret.png)
```
Diagram doesn't seem to match all the instructions above. Not in "Administrator" view, and where is the "Select a workload" button?

markstur committed:
More AWS app config screenshots and more in sync w/ instructions

The screenshot wasn't quite what the instructions said.
Also, there were a lot of instructions w/ minimal screenshots
which might have been good enough, but was really confusing if
you went looking for screen elements that were not pictured.
So more screenshots were added.

markstur committed:
Add red boxes on the AWS app deploy images
